File: projekt/projekt.java
import java.math.BigDecimal;
import java.math.BigInteger;
import java.math.MathContext;
import java.util.InputMismatchException;
import java.util.Scanner;


class Algorithm {
    public void solution(BigInteger n){
        BigDecimal Decimal_n = new BigDecimal(n);
        BigDecimal sum = Decimal_n.add(BigDecimal.ONE).sqrt(new MathContext(100));

        System.out.println("numeric sum: "+sum);
        System.out.println("symbolic sum: sqrt("+n.add(BigInteger.ONE)+")");
    }
}

public class Main {
    public static void main(String[] args) {
        BigInteger n;
        try {
            Scanner scanner = new Scanner(System.in);
            System.out.println("Enter n:");
            n = scanner.nextBigInteger();

            if (n.compareTo(BigInteger.ONE) < 0) {
                throw new Exception("n must be greater than 0");
            }
        }catch (InputMismatchException e) {
            System.out.println("n must be a number");
            return;
        }catch (Exception e) {
            System.out.println(e.getMessage());
            return;
        }

        Algorithm calculate = new Algorithm();
        calculate.solution(n);
    }
}

File: projekt/projekt.py
import math
import sys
from datetime import datetime

class Algorithm():
    def solution(self, n):
        sum = math.sqrt(n + 1)
        output=""
        output+=f"numeric sum: {sum}\n"
        output+=f"symbolic sum: sqrt({n+1})\n"
        return output

if __name__ == "__main__":
    exitcode = 0
    htmlendl = "<br>\n"
    now = datetime.now()
    fulldate = now.strftime("%d-%m-%Y %H:%M:%S")

    file_input = open("input", "r")
    file_output = open("output", "w")
    file_html = open("output.html", "w")

    file_html.write("<html><body>")
    file_html.write(f"<h1>{fulldate}</h1>\n")

    calculate = Algorithm()

    for line in file_input:
        try:
            file_html.write(f"Enter n: {line}"+htmlendl)
            n = int(line)
            
            if n < 1:
                raise Exception("n must be greater than 0")
            
            output=calculate.solution(n)
            output_html=output.replace("\n", htmlendl)

            file_output.write(output)
            file_html.write(output_html)
        except ValueError:
            file_html.write("n must be a number"+htmlendl)
            exitcode = 1
        except Exception as e:
            file_html.write(str(e)+htmlendl)
            exitcode = 1
    file_html.write("</body></html>")

    file_input.close()
    file_output.close()
    file_html.close()

    sys.exit(exitcode)
